frontend/javascripts/admin/api/jobs.ts

```diff
@@ -222,6 +224,12 @@ function startSegmentationAnnotationDependentJob(
   if (mergeSegments != null) {
     requestURL.searchParams.append("mergeSegments", mergeSegments.toString());
   }
+  if (includesProofreading != null) {
+    requestURL.searchParams.append("includesProofreading", includesProofreading.toString());
+  }
+  if (selectedBoundingBox) {
+    requestURL.searchParams.append("selectedBoundingBox", selectedBoundingBox.toString());
```

Comment:
```suggestion
    requestURL.searchParams.append("selectedBoundingBox", selectedBoundingBox.join(","));
```

should be identical, but in your variant one has to know how JS serializes number arrays. I find it better to be explicit about it.